src/metabase/public_settings.clj

```diff
@@ -60,22 +60,30 @@
   (deferred-tru "The name used for this instance of Metabase.")
   :default "Metabase")
 
-(defn uuid-nonce
-  "Getter for settings that should be set to a UUID the first time they are fetched."
-  [setting]
-  (or (setting/get-string setting)
+;; `::uuid-nonce` is a Setting that sets a site-wide random UUID value the first time it is fetched.
```

Comment:
I'm actually surprised that there's no british peeps at MB but I'm just going to note that 'nonce' is british slang for 'pedophile'

Reply:
Lol TIL. 

That aside I'm not actually convinced `nonce` is the best name for this anyway since it's actually used more than once (it's just randomly generated on first use) but I'm not sure what to name this instead. Any suggestions?

Since this name wasn't introduced in this PR (only in the diff since I upgraded it to an actual Setting type) I'm not super worried about changing it here either way.

Reply:
"singe-use-token"? thats the phrase stripe uses